src/bp/ui-shared/src/Contents/Components/Form/index.tsx

```diff
@@ -180,14 +181,30 @@ const Form: FC<FormProps> = ({
         onBlur={value => {
           update(value)
         }}
+        refValue={refValue}
         value={data}
       />
     )
   }
 
+  const getRefValue = (value, currentLang, defaultLang) => {
+    if (currentLang !== defaultLang || !value[defaultLang]) {
+      const refLang = Object.keys(value).find(key => key !== currentLang && value[key])
+
+      return refLang && value[refLang]
+    }
+
+    return value[defaultLang]
+  }
```

Comment:
This would technically not be required anymore because the reference value = default lang

Reply:
Working this way so I won't change it until translation center